API.md

```diff
@@ -468,6 +476,7 @@ MixPanel](https://permanent.atlassian.net/wiki/spaces/EN/pages/2086862849/Mixpan
 
 - Headers: Authorization: Bearer \<JWT from FusionAuth>
 - Response
+
```

Comment:
I'm guessing these whitespace changes came from an auto-formatter; what tool are you using?

Reply:
i use Prettier for formatting. should I be using something else? or deactivate it on `.md` files?

Reply:
Prettier is our formatter of choice, and I see no reason that it shouldn't be used on Markdown files. I was curious because I thought my Prettier checks should have caught this, but I realize now that my editor isn't currently running it on Markdown files; I'll change that.